crates/node/service/src/actors/sequencer/admin_api_impl.rs

```diff
@@ -0,0 +1,129 @@
+use super::SequencerActor;
+use crate::{BlockBuildingClient, Conductor, OriginSelector, SequencerAdminQuery};
+use alloy_primitives::B256;
+use kona_derive::AttributesBuilder;
+use kona_rpc::SequencerAdminAPIError;
+
+/// Handler for the Sequencer Admin API.
+impl<AB, C, OS, BB> SequencerActor<AB, C, OS, BB>
+where
+    AB: AttributesBuilder,
+    C: Conductor,
+    OS: OriginSelector,
+    BB: BlockBuildingClient,
+{
+    /// Handles the provided [`SequencerAdminQuery`], sending the response via the provided sender.
+    /// This function is used to decouple admin API logic from the response mechanism (channels).
+    pub(in crate::actors::sequencer) async fn handle_admin_query(
+        &mut self,
+        query: SequencerAdminQuery,
+    ) {
+        match query {
+            SequencerAdminQuery::SequencerActive(tx) => {
+                if tx.send(self.is_sequencer_active().await).is_err() {
+                    warn!(target: "sequencer", "Failed to send response for is_sequencer_active query");
+                }
+            }
+            SequencerAdminQuery::StartSequencer(tx) => {
+                if tx.send(self.start_sequencer().await).is_err() {
+                    warn!(target: "sequencer", "Failed to send response for start_sequencer query");
+                }
+            }
+            SequencerAdminQuery::StopSequencer(tx) => {
+                if tx.send(self.stop_sequencer().await).is_err() {
+                    warn!(target: "sequencer", "Failed to send response for stop_sequencer query");
+                }
+            }
+            SequencerAdminQuery::ConductorEnabled(tx) => {
+                if tx.send(self.is_conductor_enabled().await).is_err() {
+                    warn!(target: "sequencer", "Failed to send response for is_conductor_enabled query");
+                }
+            }
+            SequencerAdminQuery::SetRecoveryMode(is_active, tx) => {
+                if tx.send(self.set_recovery_mode(is_active).await).is_err() {
+                    warn!(target: "sequencer", is_active = is_active, "Failed to send response for set_recovery_mode query");
+                }
+            }
+            SequencerAdminQuery::OverrideLeader(tx) => {
+                if tx.send(self.override_leader().await).is_err() {
+                    warn!(target: "sequencer", "Failed to send response for override_leader query");
+                }
+            }
+        }
+    }
+
+    pub(in crate::actors::sequencer) async fn is_sequencer_active(
+        &self,
+    ) -> Result<bool, SequencerAdminAPIError> {
+        Ok(self.is_active)
+    }
+
+    pub(in crate::actors::sequencer) async fn is_conductor_enabled(
```

Comment:
In that case you (and everywhere else in that file), you can just use instead:

```suggestion
    pub(super) async fn is_conductor_enabled(
```